dev/sparktestsupport/modules.py

```diff
@@ -539,7 +539,7 @@ def __hash__(self):
 kubernetes = Module(
     name="kubernetes",
     dependencies=[],
-    source_file_regexes=["resource-managers/kubernetes/core"],
+    source_file_regexes=["resource-managers/kubernetes"],
```

Comment:
Why this change?

Reply:
Otherwise when you change something outside of kubernetes/core (like the docker files), all unit tests will be run, not just the stuff related to k8s.

I requested this change in the original PR but you guys never addressed it.

Reply:
Ah, sorry the comment was not caught by us.